mobile/src/main/java/org/openhab/habdroid/core/OnUpdateBroadcastReceiver.java

```diff
@@ -0,0 +1,48 @@
+package org.openhab.habdroid.core;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import org.openhab.habdroid.util.Constants;
+
+public class OnUpdateBroadcastReceiver extends BroadcastReceiver {
```

Comment:
Does openhab recognize this broadcast even when its not running?

Reply:
It should, the OS should create the Intent and delivers it to the app, which was "replaced" only, which then can respond to it as described in the manifest. In this case, the OnUpdateBroadcastReceiver will be called. That happens also, if the app is currently not running.